docs/developer/local-docker.md

```diff
@@ -50,6 +59,97 @@ This will launch the integration test  runner against the CI environment, using
 
 This may all be overridden.
 
+## Unit tests
+
+Unit tests require a running Narrative server in order to access Jupyter javascript files which are only available when the server is built and running.
+
+As a server, it can be accessed within a container is well, and for those who prefer Docker-based workflows, is quite convenient.
+
+- run the container as specified above. For example:
+
+```shell
+make dev-image
+ENV=ci make run-dev-image
+```
+
+Then, to run tests without authentication.
+
+```shell
+nvm use 16
```

Comment:
is there something earlier in the guide about using nvm for node management?

Reply:
Nope. You know, this document has been sitting in the repo for many years with very little change. It was basically to memorialize how I develop the Narrative with docker, rather than running it on the bare host with Python. It isn't a "blessed" Narrative workflow - as far as I know, no-one else uses it. Well, no-one ever commented on it before!

So I'm happy to make changes to it and will, but that is why it is fairly bare-bones.

Reply:
Wow, that comment was from over a year ago. I was a different person then!